nipype/algorithms/stats.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+"""
+Managing statistical maps
+"""
+from __future__ import (print_function, division, unicode_literals,
+                        absolute_import)
+import os
+import nibabel as nb
+import numpy as np
+
+from ..interfaces.base import (
+    BaseInterfaceInputSpec, TraitedSpec, SimpleInterface,
+    traits, InputMultiPath, File
+)
+from ..utils.filemanip import split_filename
+
+
+class ActivationCountInputSpec(BaseInterfaceInputSpec):
+    in_files = InputMultiPath(File(exists=True), mandatory=True,
+                              desc='input file, generally a list of z-stat maps')
+    threshold = traits.Float(1.65, usedefault=True,
+                             desc='binarization threshold. A z-value of 1.65 '
+                                  'corresponds to a two-sided test of p<.10')
+
```

Comment:
I'm wondering if it's better to leave `threshold ` without default value, so users have to really choose one....

Reply:
I don't have an opinion here.

Reply:
Makes sense, I'll update accordingly.